DSharpPlus.Rest/DiscordRestClient.cs

```diff
@@ -1128,9 +1128,10 @@ public async Task<DiscordGuildWelcomeScreen> ModifyGuildWelcomeScreenAsync(ulong
         /// </summary>
         /// <param name="invite_code">The invite code.</param>
         /// <param name="withCounts">Whether to include presence and total member counts in the returned invite.</param>
+        /// <param name="withExpiration">Whether to include the expiration date in the returned invite.</param>
         /// <returns></returns>
-        public Task<DiscordInvite> GetInvite(string invite_code, bool? withCounts = null)
-            => this.ApiClient.GetInviteAsync(invite_code, withCounts);
+        public Task<DiscordInvite> GetInvite(string invite_code, bool? withCounts = null, bool? withExpiration = null)
```

Comment:
We might as well change this to GetInviteAsync() since we're already doing a breaking change here.

Reply:
It's not a breaking change since it's an optional argument

Reply:
It technically is an ABI breaking change... I think.